### DISCUSSION TITLE
HTTP/3 support

### DISCUSSION THREAD

**Discussion by sla-te** [encode/httpx#1696] [General]
Is there any possibility to patch httpx to get HTTP/3 working, at least for the client?

**Comment by lovelydinosaur** [comment]
You're probably going to have to figure that out mostly by yourself. HTTP/3 support isn't going to be a priority for us for a while, yet.

**Reply by sla-te** [reply]
Is there anything you can recommend me or is there any other package or procedure you know of to make it work? I literally need it only for 2 requests in my app, where a webserver enforces http3 over http2.

**Reply by johtso** [reply]
Not sure what state this is in, but could be of some use.. https://www.github.com/aiortc/aioquic/tree/main/examples%2Fhttpx_client.py

**Reply by sla-te** [reply]
Yeah I have tried that but its so outdated, that it doesnt work anymore - You get exceptions etc..
